sw/device/rom_exts/manifest.md

```diff
@@ -240,19 +244,21 @@ Notes:
     used to store a checksum of the extension's data, using CRC32.
 
     Each Offset field is a 32-bit unsigned numeric value. Each Checksum field is
-    a 32-bit numeric value.
-
-    **Open Q** It makes sense, once we fully understand the alignment
-    requirements, to fill the rest of the manifest with these pairs, so we may
-    end with more than 4 of these entries in the first production Mask ROM.
-
-    These extensions will not be allocated in the initial version of the
-    ROM_EXT. Once an Extension slot has an allocated use, it will not be used
-    for any other use, for a given Manifest Image Identifier. In this way, the
-    Identifier also works as a versioning token for the manifest format.
-
+    a 32-bit numeric value. Alignment requirements for the Offset field are
+    defined when an extension pair is allocated meaning. If none is defined, the
+    Offset is byte-aligned.
```

Comment:
> If none is defined, the Offset is byte-aligned.

Would it make sense having the minimum alignment requirements for all the manifest fields to 32bit?

Reply:
I suppose you mean the alignment of the data where this offset points? Maybe it is worth rephrasing slightly to explicitly mention that the alignment talk is related to the extension data (not the manifest field)?

Reply:
Yeah, hopefully I have updated this to clarify your confusion.